targets/targets.json

```diff
@@ -7842,29 +7844,22 @@
     "NRF52840_DK": {
         "supported_form_factors": ["ARDUINO"],
         "inherits": ["MCU_NRF52840"],
-        "release_versions": ["5"],
-        "device_name": "nRF52840_xxAA",
+        "detect_code": ["1102"],
         "macros_add": [
             "CONFIG_GPIO_AS_PINRESET"
         ]
     },
     "ARDUINO_NANO33BLE": {
         "inherits": ["MCU_NRF52840"],
-        "release_versions": ["5"],
-        "device_name": "nRF52840_xxAA",
-        "features_add": ["BLE", "STORAGE"],
+        "features_add": ["STORAGE"],
         "components_remove": ["QSPIF"],
-        "components_add": ["FLASHIAP"],
         "device_has_remove": ["QSPI", "ITM"],
-        "device_has_add": ["FLASH"],
```

Comment:
this one is neither set in the `MCU_NRF52840` ?

Reply:
All three - BLE, FLASHIAP and FLASH - are already defined under MCU_NRF52840

````
"MCU_NRF52840": {
        "inherits": ["Target"],
        "components_add": ["QSPIF", "FLASHIAP"],
        "core": "Cortex-M4F",
        "macros": [
            "BOARD_PCA10056",
            "NRF52840_XXAA",
            "TARGET_NRF52840",
            "CMSIS_VECTAB_VIRTUAL",
            "CMSIS_VECTAB_VIRTUAL_HEADER_FILE=\"cmsis_nvic.h\"",
            "MBED_TICKLESS",
            "MBEDTLS_CONFIG_HW_SUPPORT",
            "WSF_MAX_HANDLERS=10",
            "MBED_MPU_CUSTOM",
            "SWI_DISABLE0",
            "NRF52_PAN_20"
        ],
        "features": ["CRYPTOCELL310", "BLE"],
        "device_has": [
            "ANALOGIN",
            "FLASH",
```